blocks/identity-block/components/social-sign-on/_children/GoogleSignIn.jsx

```diff
@@ -1,5 +1,54 @@
-import React from "react";
+import React, { useEffect } from "react";
 
-const GoogleSignIn = () => <div id="google-sign-in-button" />;
+import { Button, Icon, usePhrases } from "@wpmedia/arc-themes-components";
+
+import { SIGN_UP } from "../constants";
+
+const GoogleIcon = <Icon name="GoogleColor" />;
+
+function GoogleSignIn({ customButtons, socialSignOnIn, className }) {
+	const phrases = usePhrases();
+	const googleNotification = phrases.t("identity-block.social-google-one-tap-notification");
+
+	useEffect(() => {
+		if (customButtons) {
+			document.getElementById("custom-google-signin-btn").addEventListener("click", () =>
+				window.google.accounts.id.prompt((notification) => {
+					if (notification.isNotDisplayed() || notification.isSkippedMoment()) {
```

Comment:
It seems like google doesn't want you to use `isNotDisplayed` with FedCM roll out.
https://developers.google.
![Screenshot 2024-07-09 at 1 37 49 PM](https://github.com/WPMedia/arc-themes-blocks/assets/67028780/efa63ec6-eb9b-45cf-accc-f4537a9f58bc)

[com/identity/gsi/web/guides/fedcm-migration?s=dc#display_moment](https://developers.google.com/identity/gsi/web/guides/fedcm-migration?s=dc#display_moment)